### PR TITLE
Add br.com.zoop.zooppcserver

### PR DESCRIPTION
**Pull request by meloluan**
### Please confirm your submission meets all the criteria

- [X] Please describe the application briefly.
Our app works by facilitating financial transactions at Zoop quickly and securely using a pinpad.
Designed to provide a smooth user experience, it allows payments and financial transactions to be made without the need for constant user intervention.

- [X] The domain used for the application ID is [controlled by the application developers][appid-domain] and the [application id guidelines][appid] are followed.
- [X] I have read the and followed all the [Submission and licence requirements][reqs].
- [X] I have [built][build] and tested the submission locally.
- [X] I am an author/developer/upstream contributor of the project. If not, I contacted upstream developers about this submission. **Link:**

[appid-domain]: https://docs.flathub.org/docs/for-app-authors/requirements/#control-over-domain-or-repository
[appid]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id
[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission

### CODE REVIEW

File: br.com.zoop.zooppcserver.desktop [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Version=3.7.4
```
Comment (hfiguiere) [inline br.com.zoop.zooppcserver.desktop:2]: this value is invalid. This is NEVER the application version.

File: br.com.zoop.zooppcserver.desktop [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+Name=ZoopPcServer
+Comment=O aplicativo facilita transações financeiras em segundo plano usando um pinpad, garantindo segurança e eficiência. Ideal para comerciantes, ele oferece uma interface intuitiva e suporte para múltiplos métodos de pagamento.
```
Comment (hfiguiere) [inline br.com.zoop.zooppcserver.desktop:4]: This should be in English, and then localized.

File: br.com.zoop.zooppcserver.metainfo.xml [hunk 3]
```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>br.com.zoop.zooppcserver</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-3.0-or-later</project_license>
+  <name>ZoopPcServer</name>
+  <summary>O aplicativo facilita transações financeiras em segundo plano usando um pinpad, garantindo segurança e eficiência. Ideal para comerciantes,
```
Comment (hfiguiere) [inline br.com.zoop.zooppcserver.metainfo.xml:7]: this should be in English.

File: br.com.zoop.zooppcserver.metainfo.xml
Comment (hfiguiere) [inline br.com.zoop.zooppcserver.metainfo.xml:1]: this file doesn't validate.

File: br.com.zoop.zooppcserver.yml [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+    buildsystem: simple
+    build-commands:
+      - install -d ${FLATPAK_DEST}/bin
+      - cp launcher ${FLATPAK_DEST}/bin/
+      - cp pinpad-server.jar ${FLATPAK_DEST}/bin/
+      - chmod +x ${FLATPAK_DEST}/bin/launcher
+      - install -d ${FLATPAK_DEST}/share/applications
+      - install -Dm644 br.com.zoop.zooppcserver.desktop -t ${FLATPAK_DEST}/share/applications/
+      - install -d ${FLATPAK_DEST}/share/icons/hicolor/256x256/apps
+      - install -Dm644 br.com.zoop.zooppcserver.png -t ${FLATPAK_DEST}/share/icons/hicolor/256x256/apps/
```
Comment (hfiguiere) [inline br.com.zoop.zooppcserver.yml:25]: the image is not 256x256.... 
its size is actually non standard and smaller than the minimum 128x128

**Comment by meloluan** [comment]
> What the relation with #5607 ?

PR #5607 should be closed.

File: br.com.zoop.zooppcserver.yml [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+  - name: ZoopPcServer
+    buildsystem: simple
+    build-commands:
+      - install -d ${FLATPAK_DEST}/bin
+      - cp launcher ${FLATPAK_DEST}/bin/
+      - cp pinpad-server.jar ${FLATPAK_DEST}/bin/
+      - chmod +x ${FLATPAK_DEST}/bin/launcher
+      - install -d ${FLATPAK_DEST}/share/applications
+      - install -Dm644 br.com.zoop.zooppcserver.desktop -t ${FLATPAK_DEST}/share/applications/
+      - install -d ${FLATPAK_DEST}/share/icons/hicolor/256x256/apps
```
Comment (hfiguiere) [inline br.com.zoop.zooppcserver.yml:24]:
```suggestion
```

File: br.com.zoop.zooppcserver.yml [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+  - --env=JAVA_HOME=/app/jre
+modules:
+  - name: ZoopPcServer
+    buildsystem: simple
+    build-commands:
+      - install -d ${FLATPAK_DEST}/bin
+      - cp launcher ${FLATPAK_DEST}/bin/
+      - cp pinpad-server.jar ${FLATPAK_DEST}/bin/
+      - chmod +x ${FLATPAK_DEST}/bin/launcher
+      - install -d ${FLATPAK_DEST}/share/applications
```
Comment (hfiguiere) [inline br.com.zoop.zooppcserver.yml:22]:
```suggestion
```

File: br.com.zoop.zooppcserver.yml [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+    sources:
+      - type: file
+        path: launcher
+      - type: file
+        path: pinpad-server.jar
+      - type: file
+        path: br.com.zoop.zooppcserver.desktop
+      - type: file
+        path: br.com.zoop.zooppcserver.png
+sdk-extensions:
```
Comment (hfiguiere) [inline br.com.zoop.zooppcserver.yml:36]: this should be above finish args.

File: pinpad-server.jar
Comment (hfiguiere) [inline pinpad-server.jar:1]: this DOES NOT belong here.

File: br.com.zoop.zooppcserver.metainfo.xml
Comment (hfiguiere) [reply br.com.zoop.zooppcserver.metainfo.xml:1]: also it is never installed.

File: br.com.zoop.zooppcserver.yml [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+  - --env=PATH=/app/bin:/usr/bin:/app/jre/bin
+  - --env=JAVA_HOME=/app/jre
+modules:
+  - name: ZoopPcServer
+    buildsystem: simple
+    build-commands:
+      - install -d ${FLATPAK_DEST}/bin
+      - cp launcher ${FLATPAK_DEST}/bin/
+      - cp pinpad-server.jar ${FLATPAK_DEST}/bin/
+      - chmod +x ${FLATPAK_DEST}/bin/launcher
```
Comment (hfiguiere) [inline br.com.zoop.zooppcserver.yml:21]:
```suggestion
      - install -Dm755 launcher -t ${FLATPAK_DEST}/bin/
      - install -D pinpad-server.jar -t ${FLATPAK_DEST}/bin/
```

File: br.com.zoop.zooppcserver.metainfo.xml [hunk 9]
```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>br.com.zoop.zooppcserver</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-3.0-or-later</project_license>
```
Comment (hfiguiere) [inline br.com.zoop.zooppcserver.metainfo.xml:5]: I don't see any indication that this is the license since there is not even a link to the source code.

Comment (meloluan) [reply br.com.zoop.zooppcserver.metainfo.xml:5]: Is it necessary to be open source? I need to remove that license there.

Comment (hfiguiere) [reply br.com.zoop.zooppcserver.metainfo.xml:5]: It must be accurate.

**Comment by hfiguiere** [comment]
> > What the relation with #5607 ?
> 
> PR #5607 should be closed.

you are not the same submitter.

**Pull request closed without merging**